script.mjs

```diff
@@ -69,20 +70,17 @@ window.read_saved_url = function (callback) {
 };
 
 window.save_clipboard = function (text) {
-    new Promise(async (resolve) => {
-        try {
-            const clipboard = window.navigator.clipboard;
-            if (clipboard) {
-                await clipboard.writeText(text);
-            } else {
-                alert("Clipboard is not available.");
-            }
-        } catch (e) {
-            console.error("could not store to clipboard", e);
-        } finally {
-            resolve();
-        }
-    });
+    const el = document.createElement("textarea");
+    el.value = text;
+    el.setAttribute("readonly", "");
+    // To not make it appear on the screen.
+    el.style.position = "absolute";
+    el.style.left = "-9999px";
+
+    document.body.appendChild(el);
+    el.select();
+    document.execCommand("copy");
```

Comment:
[`document.execCommand()`](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand) is deprecated, otherwise I would have used it. :)

Reply:
It's still supported by all browsers though. :3

Reply:
Odd, for me it does not work. Maybe this method needs HTTPS by now, too?

But I tested it. My previous implementation does work in production: <https://k6i.de/dist/>. I would rather use the newer API.

Reply:
You can't use the copy button in rustdoc on local docs? But anyway, removing the change.

Reply:
Odd, yes, there it does indeed work.